### DISCUSSION TITLE
关于探求译词标准大讨论

### DISCUSSION THREAD

**Discussion by ahdg6** [Tartaros-Gamer/GriefDefenderSinicization#5] [General]
以下仅表达我的观点
Basic = 基础   --- 考虑到翻译为普通的话，会和 common 冲突
Context = 情境
Bonus = 额外奖励
Accrued = 累积
在 economy-block 情境下，领地方块数应该改为领地格数

**Comment by pys078** [comment]
参考 luckperms ，context意思大概为权限的补充内容，情境一词可能难以理解

**Reply by ahdg6** [reply]
LuckPerms 中就是翻译为 情境 的

**Reply by ahdg6** [reply]
引申一下词典里的释义
“情景”的“景”是具体、直观和吸引人的，指具体场合的情形，景象。“情境”的“境”是指构成和蕴涵在情景中的那些相互交织的因素及其相互之间的关系。
后者更适合描述 GD 在处理多种因素 source target 之类的情形

**Reply by pys078** [reply]
那就译作情境吧
